Tests/Integration/IntegrationTestData.swift

```diff
@@ -11,8 +11,18 @@ let checkEmailTryCount = 6
 
 let unregisteredUserEmail = "unregistered-test-user@passage.id"
 let registeredUserEmail = "ricky.padilla+user01@passage.id"
+
 let magicLinkAppId = "czLTOVFIytGqrhRVoHV9o8Wo"
 let magicLinkRegisteredUserEmail = "blayne.bayer@passage.id"
+let magicLinkUnactivatedId = "ioM1TTG0eiWMrOq9FA7X5zMN"
+
+let otpAppId = "pTBeTnbvm1z3U6hznMTD33Es"
```

Comment:
the AppId is different than what we use on the android side tests. What is name of this app in console?